sdk/src/plugins/ratePlugin/rateMock/RateMock.ts

```diff
@@ -55,4 +55,17 @@ export class RateMockPlugin implements IRateMockPlugin {
             })
             .instruction();
     }
+
+    async initialize() {
```

Comment:
can we create an "initialize" method on the interface level? for both plugin families

Reply:
Silly mistake. Added the initialise function to interface as well.

Reply:
no no, the plugin initialization should remain outside the vyper logic, for better logic decoupling in future